modules/core/src/Storage/SQLPermanentStorage.php

```diff
@@ -33,7 +33,7 @@ public function __construct(string $name, Configuration $config = null)
             $config = Configuration::getInstance();
         }
 
-        $datadir = $config->getPathValue('datadir', 'data/') ?: 'data/';
+        $datadir = $config->getPathValue('datadir', sys_get_temp_dir()) ?: sys_get_temp_dir();
```

Comment:
data/ is different from tmp (the former is expected to be persistent in ssp). So if you default to /tmp you e.g. lose your sql "permanent" storage after reboot. Maybe better to default to not working than by default storing permanent things in /tmp 'accidentally'?

Reply:
Agreed. Will change to throw a `CriticalConfigurationException` when unset.